sbi/inference/potentials/score_fn_iid.py

```diff
@@ -109,7 +118,7 @@ def __call__(
 class FNPEScoreFunction(IIDScoreFunction):
```

Comment:
I am a bit concerned about this name because it is so close to `FMPE`. can we think of a different name here? `FactorizedNPE`?